src/storage/types.rs

```diff
@@ -34,6 +34,19 @@ impl Key {
         encoded.write_u64::<BigEndian>(ts).unwrap();
         Key(encoded)
     }
+
+    /// Returns a key that can be used for `seek` next key.
+    pub fn seek_key(&self) -> Key {
+        self.encode_ts(u64::max_value())
+    }
+
+    /// Returns a key that can be used for `reverse_seek` next key.
+    pub fn reverse_seek_key(&self) -> Key {
+        let mut encoded = self.0.clone();
+        let len = encoded.len();
+        encoded[len - 1] -= 1u8;
```

Comment:
@siddontang Last character will not be 0x00 since the key is always encoded, and when a key is encoded, last byte will be the PADDING_MARKER, which is not less than `0xff - 8`.